### PR TITLE
Doc 11369 - Update docs to include _system scope: overview UI, cbstats, and REST

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Review by stevewatanabe — APPROVED** [review]

**Review by stevewatanabe — APPROVED** [review]

File: modules/cli/pages/cbstats/cbstats-collections.adoc [hunk 1]
```diff
@@ -192,22 +116,27 @@ Each data key is concluded with a colon, and the associated value is presented t
 
 Within the output, each successive group of lines presents information on one of the collections for the specified bucket, on the specified node.
 
-The first collection is identified, on each of the first nine lines, as `0x0:0x0`; and, on line five, as the `&#95;default` collection.
+The first collection is identified by `name` as the `&#95;default` collection.
 
-Line one displays, in _bytes_, the total memory (across active vBuckets) used by this collection (here, `22137956`).
-Line two displays the total disk space occupied (across active vBuckets) by this collection for the specified bucket, on the specified node: this total (here, `2161511`) is, again, expressed in _bytes_.
+The `collections_mem_used` line displays, in _bytes_, the total memory (across active vBuckets) used by this collection.
+The `data_size` two displays the total disk space occupied (across active vBuckets) by this collection for the specified bucket, on the specified node: this total is, again, expressed in _bytes_.
```
Comment (stevewatanabe) [inline modules/cli/pages/cbstats/cbstats-collections.adoc:122]: I think "two" can be omitted.

**Review by stevewatanabe — COMMENTED** [review]
On line 122, "The `data_size` two displays" should be The `data_size` displays

**Review by stevewatanabe — COMMENTED** [review]
Looks good

File: modules/cli/pages/cbstats/cbstats-collections.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/cli/pages/cbstats/cbstats-collections.adoc:122]: Thanks, will fix.

**Pull request merged**